samples/client/petstore/python/test/test_animal.py

```diff
@@ -5,19 +5,21 @@
 
     This spec is mainly for testing Petstore server and contains fake endpoints, models. Please do not use this for any other purpose. Special characters: \" \\  # noqa: E501
 
-    OpenAPI spec version: 1.0.0
+    The version of the OpenAPI document: 1.0.0
     Generated by: https://openapi-generator.tech
 """
 
 
 from __future__ import absolute_import
 
 import unittest
+import datetime
+from io import StringIO
```

Comment:
What if we delete this import and have bytes examples use this?
`var = bytes(b'blah')`
Note: this will work for python 2.7 and later